data/core/units/saurians/Spearthrower.cfg

```diff
@@ -0,0 +1,94 @@
+#textdomain wesnoth-units
+[unit_type]
+    id=Saurian Spearthrower
+    name= _ "female^Saurian Spearthrower"
+    race=lizard
+    gender=female
+    image="units/saurians/spearthrower/spearthrower.png"
+    profile="portraits/saurians/skirmisher.webp"
+    hitpoints=39
+    movement_type=lizard
+    movement=7
+    experience=55
+    level=2
+    alignment=chaotic
+    advances_to=Saurian Javelineer
+    cost=23
+    usage=mixed fighter
+    description= _ "Some saurian warriors with a particularly good aim prefer to aid their allies from the back lines rather than to infiltrate the enemy formations. Their deadly spear throwing skills allow them to support other saurian warriors from afar."
+    die_sound=hiss-die.wav
+    [attack]
+        name=spear
+        description=_"spear"
+        range=melee
+        type=pierce
+        damage=6
```

Comment:
Same as above, I feel like a greater difference between melee and ranged would be better.

Reply:
@nemaara I think that might have some slight issues that we might want to consider:

* Their melee damage will be below the Skirmisher's. Leaving it at 6x3 has a nice parallel with the Ambusher, as the Spearthrower currently deals the same damage per hit with one less strike.
* 9x3 is already above the Oracle's damage. 10x3 will start to rival Drakes, despite a lower experience requirement.

It's not necessarily wrong, I just felt like this unit could be a mixed fighter with a ranged preference that slowly advances further from the original unit archetype into an archer. If you still think the attacks should be changed, I can adjust them as you've proposed and change the unit usage to archer.

Reply:
That's fine, leave it then.